pylinted code

ccialek opened the pull request:
Brought pylint score from 6.00 to 9.52! Changed variable name fq to out, tabbing for metadata sections, removed discard argument form fastq2qseq, made new lines to shorten, changed ==True to is True

ccialek committed:
pylinted code

brought pylint score from 6.00 to 9.52! Changed variable name fq to out, tabbing for metadata sections, removed discard argument form fastq2qseq, made new lines to shorten, changed ==True to is True

biokcb reviewed (changes requested):
can you also remove the unused files?

ccialek committed:
removed extra files

removed fastq2qseq.py, qseq2fastq.py, qseq_fastq_readin_export.py, and the testdata folder and its entire contents
